CSS/main.css

```diff
@@ -26,6 +41,49 @@ body {
     font-size: 3rem;
 }
 
+/* Theme buttons div */
+.flexrow-container {
+    align-items: center;
+    align-self: flex-end;
+    display: flex;
+    justify-content: space-around;
+    margin-right: 3%;
+}
+
+.flexrow-container > p {
+    font-size: 1rem;
+    font-weight: 600;
+}
+
+.theme-selector {
+    border: 2px solid #F2E5CB;
```

Comment:
Minor enhancement:
`border: 1px solid #d1dae3;`

Reply:
I agree, it looks better like this! 👍🏻